docs/lib/geo/menu.json

```diff
@@ -2,8 +2,8 @@
   "title": "Geo (Developer Preview)",
   "items": [
     "getting-started",
+    "existing-resources",
     "maps",
-    "search",
-    "existing-resources"
+    "search"
```

Comment:
I don't believe this is what we wanted to do or is the correct flow anyways. "Use Existing Resources" is an alternative flow described in the getting-started page so as such it should be linked from the getting started guide such as 

* Automated provisioning
  * CLI stuff
* Manual setup
  * link to use existing-resources

Reply:
This was a change requested by @hdworld11